agent_non_root_condition.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+KEYLIME_GROUP=tss
```

Comment:
Can we change this to set only if not previously set? That would allow different packagers to use different users and groups if they'd like. I imagine most will use "keylime" as the user, but might have different groups.

Something like:

```
KEYLIME_GROUP=${KEYLIME_GROUP:-'tss'}
KEYLIME_USER=${KEYLIME_USER:-'keylime'}

Reply:
absolutely, just did it exactly it.